### PR TITLE
Remove Weapons_GetOffsets and add config support for pickup sounds

### PR DESCRIPTION
**Pull request by haxtonsale**

### CODE REVIEW

File: addons/sourcemod/scripting/szf/weapons.sp [hunk 1]
```diff
@@ -139,25 +137,12 @@ void Weapons_ReplaceEntityModel(int ent, int index)
...
-	angles[2] = wep.flOffsetAngles[2];
-}
-
 // -----------------------------------------------------------
 public bool Weapons_OnPickup_Health(int client)
 {
 	if (GetClientHealth(client) < SDK_GetMaxHealth(client))
 	{
 		SpawnPickup(client, "item_healthkit_full");
-		EmitSoundToClient(client, "items/smallmedkit1.wav");
```
Comment (FortyTwoFortyTwo) [inline addons/sourcemod/scripting/szf/weapons.sp]: need to readd sound to config

File: addons/sourcemod/configs/szf/weapons.cfg [hunk 2]
```diff
@@ -3,13 +3,14 @@
...
-		// "offset_origin"	Entity origin position offset, only works for pickups
-		// "offset_angles"	Entity angles offset, only works for pickups
+		// "rarity"			Weapon's rarity ("common", "uncommon", "rare", "pickup").
+		// "attrib"			Weapon's attributes, does nothing for pickups.
+		// "text"			Text that appears in chat after picking up a weapon, does nothing for pickups.
+		// "color"			Weapon's worldmodel color.
+		// "callback"		On pickup callback (function must be public and be implemented in the plugin).
+		// "offset_origin"	Entity origin position offset, only works for pickups.
+		// "offset_angles"	Entity angles offset, only works for pickups.
+		// "sound_pickup"	Pickup sound.
```
Comment (FortyTwoFortyTwo) [inline addons/sourcemod/configs/szf/weapons.cfg]: really just "sound" should be enough and not restricted to pickups, don't see where else to use sound excluding voicelines, which that will be another time in seperate config with other sounds.

File: addons/sourcemod/scripting/szf/weapons.sp [hunk 3]
```diff
@@ -168,23 +153,20 @@ public bool Weapons_OnPickup_Health(int client)
 public bool Weapons_OnPickup_Ammo(int client)
 {
 	SpawnPickup(client, "item_ammopack_full");
-	EmitSoundToClient(client, "ui/item_heavy_gun_pickup.wav");
```
Comment (FortyTwoFortyTwo) [inline addons/sourcemod/scripting/szf/weapons.sp]: same thing

File: addons/sourcemod/scripting/szf/weapons.sp (hunk 1 above)
Comment (haxtonsale) [reply addons/sourcemod/scripting/szf/weapons.sp]: this(or the similar) sound already gets played when player picks up the medkit entity

**Comment by haxtonsale** [comment]
Done

**Pull request merged**